src/management-ui/page/courses/list.html

```diff
@@ -6,22 +6,21 @@ <h1 class="heading-large">Courses</h1>
 
                 <table>
                     <thead>
-                    <tr>
-                        <th scope="col">Title</th>
-                        <th scope="col"></th>
-                    </tr>
+                        <tr>
+                            <th scope="col">Title</th>
+                            <th scope="col"></th>
+                        </tr>
                     </thead>
                     <tbody>
                         {{#each courses as course, i}}
-                            <tr>
-                                <th scope="row">
-                                    <a href="/courses/{{course.uid}}/edit">{{course.title}}</a>
-                                </th>
-                                <td>
-                                    <!-- FIXME: not portable -->
-                                    <a href="http://lpg.dev.cshr.digital:3001/courses/{{course.uid}}/">Review</a>
-                                </td>
-                            </tr>
+                        <tr>
+                            <th scope="row">
+                                <a href="/courses/{{course.uid}}/edit">{{course.title}}</a>
+                            </th>
+                            <td>
+                                <a href="//{{config.UI_SERVER}}/courses/{{course.uid}}/">Review</a>
```

Comment:
This was a hack to let me easily check things. It won't make it into private beta and it won't work once we add auth to the management ui. If we actually want to allow preview we'd need to preview it through the management ui and block xapi events etc.. tldr: I don't think there's an actual need for configuring the UI host or necessarily exposing config to templates.